### ISSUE TITLE
Main screen: click on daily/hourly items doesn't work on screen readers

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#1648]
See `AbsDailyTrendAdapter` and `AbsHourlyTrendAdapter`
